docs/use-cases/sensor-data-query.md

```diff
@@ -0,0 +1,323 @@
+            'data.readings.distance': { $gt: .2 } } },
+        { $count: 'numStanding' }
+    ] )
+[ { numStanding: 215 } ]
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+{{< /tablestep >}}
+{{< /table >}}
+
+{{< alert title="Tip" color="tip" >}}
+If you use a data field that is named the same as a [reserved SQL keyword](https://en.wikipedia.org/wiki/List_of_SQL_reserved_words), such as `value` or `position`, you must escape that field name in your query using backticks ( <file>\`</file> ).
+For example, to query against a field named `value` which is a subfield of the `data` field in the `readings` collection, you would use:
+
+```mongodb {class="command-line" data-prompt=">"}
+select data.`value` from readings
+```
+
+See the [MongoDB Atlas Documentation](https://www.mongodb.com/docs/atlas/data-federation/query/sql/language-reference/#compatability-and-limitations) for more information.
+
+{{< /alert >}}
+
+For information on connecting to your Atlas instance from other MQL clients, see the MongoDB Atlas [Connect to your Cluster Tutorial](https://www.mongodb.com/docs/atlas/tutorial/connect-to-your-cluster/).
+
+<!-- markdownlint-disable-file MD001 -->
+
+{{% expand "Need to query by date? Click here." %}}
+
+##### Query by date
+
+When using MQL to query your data by date or time range, you can optimize query performance by avoiding the MongoDB `$toDate` expression, using the [BSON `date` type](https://www.mongodb.com/docs/manual/reference/bson-types/#date) instead.
+
+For example, you could use the following query to search by a date range in the `mongosh` shell, using the JavaScript `Date()` constructor to specify an explicit start timestamp, and use the current time as the end timestamp:
+
+```mongodb {class="command-line" data-prompt=">"}
+// Switch to sensorData database:
+use sensorData
+
+// Set desired start and end times:
+const startTime = new Date('2024-02-10T19:45:07.000Z')
+const endTime = new Date()
+
+// Run query using $match:
+db.readings.aggregate(
+    [
+        { $match: {
+            time_received: {
+                $gte: startTime,
+                $lte: endTime }
+        } }
+    ] )
+```
+
+{{% /expand%}}
+
+## Search data with the Python SDK
+
+{{< table >}}
+{{< tablestep link="/build/program/#requirements">}}
+**1. Install the Python SDK**
+
+For macOS (both Intel x86_64 and Apple Silicon) or Linux (x86, aarch64, armv6l), run the following commands:
+
+```sh {class="command-line" data-prompt="$"}
+python3 -m venv .venv
+source .venv/bin/activate
+pip install viam-sdk
+```
+
+{{< /tablestep >}}
+{{< tablestep link="/cli/#organizations">}}
+**2. Create an API key**
+
+To access your machines using the Python SDK, you must use an API key:
+
+```sh {class="command-line" data-prompt="$"}
+viam organizations api-key create --org-id <org-id> --name my-api-key
+```
+
+{{< /tablestep >}}
+{{< tablestep link="/appendix/apis/data-client/">}}
+**3. Use the API key with the `data_client`**
```

Comment:
I think this could use more instructions about what an api key / key ID is and how to get them from app.viam.com -- I guess create command returns this so might not be necessary but maybe just a sentence about what it is/link to permissions page?